app/controllers/reports/hic/inventories_controller.rb

```diff
@@ -8,8 +8,11 @@ module Reports
   class Hic::InventoriesController < Hic::BaseController
     def show
       @inventories = GrdaWarehouse::Hud::Inventory.joins(:project).
-        where(i_t[:InventoryStartDate].gt((Time.now.beginning_of_year - 1.year).to_date).or(i_t[:InventoryStartDate].eq(nil))).
-        where(Project: { computed_project_type: PROJECT_TYPES }).
+        merge(GrdaWarehouse::Hud::Project.viewable_by(current_user)).
+        merge(GrdaWarehouse::Hud::Project.with_hud_project_type(PROJECT_TYPES)).
+        where(
+          i_t[:InventoryStartDate].gt((Time.now.beginning_of_year - 1.year).to_date).or(i_t[:InventoryStartDate].eq(nil)),
```

Comment:
In this particular case (with `.beginning_of_year`) it will only make a difference a few hours a year, but yes, it appears that would be best practice. 
https://thoughtbot.com/blog/its-about-time-zones#do-use